packages/bpk-docs/src/helpers/tokens-helper.js

```diff
@@ -16,14 +16,12 @@
  * limitations under the License.
  */
 
-import union from 'lodash/union';
-import kebabCase from 'lodash/kebabCase';
-
+import _ from 'lodash';
```

Comment:
I don't know what is our policy about importing the whole lib rather than individual functions but this means every project using backpack will get the whole lodash lib included too. I see it is necessary for the `chain` logic bellow but this can be written using the functional modules (e.g `lodash/fp/reduce`) that support currying by default. It would become something like this:

`flow(reduce(...), sortBy(...), keyBy(...))(keys || Object.keys(tokens)`

With the added benefit that we will be importing only what's being used.

Reply:
Good point Tiago, generally we care about this, but this is the docs page which is not consumed by so it's not as important in this case.

Reply:
🤦‍♂️ , forgot it was the doc site. although a smaller bundle size is always better 😀